sandbox/urls.py

```diff
@@ -12,7 +12,6 @@
 
 from cms.sitemaps import CMSSitemap
 
-from richie.apps.core.urls import urlpatterns as core_urlpatterns
```

Comment:
In commit message: 

`s/backend to not need anymore/backend do not need anymore/`